eixampledb-core/src/main/java/com/eixampledb/core/impl/EixampleDbMapImplementation.java

```diff
@@ -17,6 +20,22 @@
 public class EixampleDbMapImplementation implements EixampleDbBackend {
 
     private final ConcurrentMap<String, EixampleDbEntry> map = new ConcurrentHashMap<>();
+    // For now files are saved to "log.json"
+    String now = new SimpleDateFormat("yyyy.MM.dd.HH.mm.ss").format(new Date());
+    private final Snapshot snapshot = new Snapshot("log" + ".json", map);
```

Comment:
File name should be configurable.

Reply:
See https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-external-config.html